lib/graphql_devise/engine.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require 'graphql_devise/rails/routes'
```

Comment:
Not sure if doing it on the engine.rb file has a benefit? Perhaps we are sure RoutesMapper is defintively loaded when we include our module?

Reply:
Calling  include on  `ActionDispatch::Routing::Mapper ` is the new way of doing this, since that happens in the entry point of the gem the module will be include as soon as the gem is loaded.
I think this also opens the possibility of supporting other frameworks besides Rails using the adapter pattern as described in [Zeitweirk readme](https://github.com/fxn/zeitwerk#use-case-the-adapter-pattern)  (but not sure just yet :sweat_smile: )

Reply:
Right, I guess that as long the include happens after requiring rails, this should work